peer/brontide.go

```diff
@@ -2938,7 +2938,7 @@ func (p *Brontide) restartCoopClose(lnChan *lnwallet.LightningChannel) (
 		})
 
 	// An error other than ErrNoShutdownInfo was returned
-	case err != nil && !errors.Is(err, channeldb.ErrNoShutdownInfo):
+	case !errors.Is(err, channeldb.ErrNoShutdownInfo):
```

Comment:
I think the nilness check is still needed here as `!errors.Is(err, ...)` is true if `err == nil`.

Reply:
I was a bit surprised too but then realized the `err == nil` case is caught above in the `case err == nil:`.

Reply:
Indeed!